### ISSUE TITLE
[BUG] CUTLASS Profiler error when --batch_count > 1 and  --gemm_kind=batched

### ISSUE THREAD

**Issue by lixhk** [NVIDIA/cutlass#900]
I buld cutlass  on RTX3070, with follow cmake:
```
cmake ..  -DCUTLASS_NVCC_ARCHS=80  -DCUTLASS_LIBRARY_KERNELS=all   
make cutlass_profiler  -j16
```

When I run  follow cmd:
```
./tools/profiler/cutlass_profiler --operation=Gemm --gemm_kind=batched --batch_count=1 --m=32 --n=512 --k=32 --A=f16:col --B=f16:col --C=f16:col --alpha=1 --beta=0   --vesrbose=true  --output=zz  
```    
There is some choices in  zz.gemm.csv 

But when I set `batch_count=8` like follow cmd:
``` 
./tools/profiler/cutlass_profiler --operation=Gemm --gemm_kind=batched --batch_count=8 --m=32 --n=512 --k=32 --A=f16:col --B=f16:col --C=f16:col --alpha=1 --beta=0   --vesrbose=true  --output=zz
```    
is not run , the Disposition: Failed , and nothing in  zz.gemm.csv

```
igs@igs:~/cutlass-master/build$ ./tools/profiler/cutlass_profiler --operation=Gemm --gemm_kind=batched --batch_count=8 --m=32 --n=512 --k=32 --A=f16:col --B=f16:col --C=f16:col --alpha=1 --beta=0   --vesrbose=true  --output=zz

=============================
  Problem ID: 1

        Provider: CUTLASS
   OperationKind: gemm
       Operation: cutlass_simt_hgemm_256x128_8x2_nn_align1

          Status: Success
    Verification: ON
     Disposition: Failed

          cuBLAS: Not run

       Arguments: --gemm_kind=universal --split_k_mode=serial --m=32 --n=512 --k=32 --A=f16:column --B=f16:column --C=f16:column  \
                  --alpha=1 --beta=0 --split_k_slices=1 --batch_count=8 --op_class=simt --accum=f16 --cta_m=256 --cta_n=128  \
                  --cta_k=8 --stages=2 --warps_m=4 --warps_n=2 --warps_k=1 --inst_m=1 --inst_n=1 --inst_k=1 --min_cc=50  \
                  --max_cc=1024

           Bytes: 540672  bytes
           FLOPs: 8650752  flops
           FLOPs/Byte: 16


Wrote results to 'zz.gemm.csv'
```

Why ? 
BTW, I want to realize  `GemmStridedBatched`  by cutlass ( get best kernel by cutlass_profiler on some m,n,k  (fp32, fp16) ),  Can you give me some advice ?

**Comment by jackkosaian** [comment]
Hi, @lixhk.

What commit of CUTLASS are you using?

We recently fixed a bug with using `batch_count > 1` in the profiler: see #873 and the fix in #883.

Are you still seeing this issue after #883?

**Comment by lixhk** [comment]
@jackkosaian   
version:  CUTLASS 2.11    CUDA 11.4     RTX3070

**Comment by jackkosaian** [comment]
Thank you for clarifying. That version of CUTLASS has the bug outlined in #873 and fixed in #883. Therefore, you are likely facing the same bug.

To fix this, you can either upgrade to the current version of CUTLASS or apply the patch from #883 to your local CUTLASS repository.

**Comment by lixhk** [comment]
@jackkosaian  OKay, thanks for your advice, I will try it, and verify.

**Comment by hwu36** [comment]
i think `-DCUTLASS_NVCC_ARCHS=80` should be 86

**Comment by lixhk** [comment]
> Thank you for clarifying. That version of CUTLASS has the bug outlined in #873 and fixed in #883. Therefore, you are likely facing the same bug.
> 
> To fix this, you can either upgrade to the current version of CUTLASS or apply the patch from #883 to your local CUTLASS repository.

Yes, I  try it and it works, Thanks

**Comment by mnicely** [comment]
@lixhk can we close this issue?
